misc/Tampermonkey/README.md

```diff
@@ -1,21 +1,29 @@
-#Tampermonkey Script
-The Tampermonkey script part
+# Tampermonkey Script
 
-## Description
-When you create Web or Sitemap sources in the Coveo platform, you can add Web scraping rules to retrieve or to exclude content from the web pages it will crawl.
-This Tampermonkey script helps with testing your web scraper rules.
-You will be able to test your rules and verify which metadata will be extracted from the page without having to re-index your source.
+The Tampermonkey script is no longer up-to-date. It is recommended to rather use the more convenient and up-to-date Google Chrome extension also included in this repo (see Chrome extension [README](../chrome_extension/README.md)).
```

Comment:
> The Tampermonkey script is no longer up-to-date.

Should we use something like "The Tampermonkey script is now deprecated." instead?

I feel like "no longer up-to-date" means we are too lazy update it. And "deprecated" means we won't update it.

Reply:
Yes, much better. Fixing it.